saas/views/download.py

```diff
@@ -421,3 +424,273 @@ def queryrow_to_columns(self, record):
             self.encode(transaction.dest_unit),
             self.encode_descr(transaction)
         ]
+
+class BalancesMetricsDownloadView(CSVDownloadView, BalancesMetricsMixin):
+    basename = 'balancesmetrics'
+
+    headings = [
+        'Date',
+        'Income',
+        'Backlog',
+        'Receivable',
+    ]
+
+    def get_headings(self):
+        return self.headings
+
+    def get_queryset(self):
+        request = Request(self.request)
+        results, unit = self.get_data(request=request)
+
+        consolidated_data = {}
```

Comment:
What is the purpose of this code block? It looks like a duplicate of some code in the `retrieve_metrics` mixin.